python/opentrons_ot3_firmware/messages/payloads.py

```diff
@@ -83,6 +83,7 @@ class AddToMoveGroupRequestPayload(MoveGroupRequestPayload):
 class AddLinearMoveRequestPayload(AddToMoveGroupRequestPayload):
```

Comment:
We'll need to add the ack id mapping in here to `MoveCompletedPayload` in this file as well

Reply:
are you referring to `request_stop_condition` inside `AddLinearMoveRequest` ? Or is there another parameter for ack_id mapping